scene/gui/tree.cpp

```diff
@@ -347,6 +350,48 @@ void TreeItem::set_collapsed(bool p_collapsed) {
 	tree->emit_signal("item_collapsed", this);
 }
 
+void TreeItem::set_collapsed_recursive(bool p_collapsed, bool p_ignore_active, bool p_skip_self) {
+
+	if (!tree) {
+		return;
+	}
+
+	if (p_skip_self || (p_ignore_active && this->is_active())) {
+		// Ignore current item, collapse recursively all children.
+		TreeItem *c = get_children();
+		while (c) {
+			c->set_collapsed_recursive(p_collapsed, p_ignore_active, false);
+			c = c->get_next();
+		}
+	} else {
+		// Set collapsed state on self and all children recursively.
+		const Variant arg_collapsed[] = { Variant(p_collapsed) };
+		const Variant *args[] = { arg_collapsed };
+		Callable::CallError error;
+		call_recursive("set_collapsed", args, 1, error);
+	}
+}
+
+void TreeItem::toggle_collapsed_all_descendants() {
+	if (is_collapsed()) {
+		// Expand recursively.
```

Comment:
Remove the comment please. The function below tells us that

Reply:
Will do!